EngineeringModel/ViewModels/PublicationBrowser/PublicationBrowserViewModel.cs

```diff
@@ -593,8 +593,8 @@ private void UpdatePublications()
             {
                 this.AddPublicationRowViewModel(publication);
             }
-
-            this.Publications.Sort((o1, o2) => o1.Index.CompareTo(o2.Index));
+            
+            this.Publications.Sort((o1, o2) => o2.PublicationDate.CompareTo(o1.PublicationDate));
```

Comment:
also possible to do in xaml `<dxg:GridColumn FieldName="Name" SortOrder="Ascending"/>` might consider as its lighter.

Reply:
indeed, but sorting in the VM was incorrect, so fixed it this way